runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultIllumina.java

```diff
@@ -36,10 +36,24 @@
 
   private static final Logger log = LoggerFactory.getLogger(DefaultIllumina.class);
 
+  private static final XPathExpression RUN_COMPLETION_STATUS_EXPRESSION = compileXPath("//CompletionStatus")[0];
+
   public static DefaultIllumina create(Builder builder, ObjectNode parameters) {
     return new DefaultIllumina(builder);
   }
 
+  private static Optional<HealthType> getHealth(Document document) {
+    try {
+      switch ((String) RUN_COMPLETION_STATUS_EXPRESSION.evaluate(document, XPathConstants.STRING)) {
+      case "CompletedAsPlanned":
+        return Optional.of(HealthType.Completed);
+      }
+    } catch (XPathExpressionException e) {
+      log.error("Failed to evaluate completion status", e);
+    }
+    return Optional.empty();
+  };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)